src/devices/Fsr408/Fsr408.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Diagnostics;
+
+namespace Iot.Device.Fsr408
+{
+    public class Fsr408 : IDisposable
+    {
+        private int _pinNumber = 0;
+
+        private Mcp3008.Mcp3008 _adcConverter;
+        public int Resistance { get; set; } = 10000; // 10k ohm
+        public int PowerSupplied { get; set; } = 5000;  // 5 mV
+
+        public Fsr408(Mcp3008.Mcp3008 adcConverter)
+        {
+            _adcConverter = adcConverter;
+        }
+
+        public Fsr408(int pinNumber, Mcp3008.Mcp3008 adcConverter)
+        {
+            _adcConverter = adcConverter;
+            _pinNumber = pinNumber;
+        }
+
+        public int Read()
+        {
+            return _adcConverter.Read(_pinNumber);
+        }
+
+        public int ReadVoltage()
+        {
+            return CalculateVoltage(_adcConverter.Read(_pinNumber));
+        }
+
+        public int CalculateVoltage(int readValue)
+        {
+            // Mcp3008 analog voltage reading ranges from 0 to 1023 (10 bit) 
+            // mapping it to corresponding milli voltage 
+            // which ranging from 0mV to PowerSupplied mV 
+            return PowerSupplied * readValue / 1023;
+        }
+
+        public int ReadFsrResistance()
+        {
+            return CalculateFsrResistance(CalculateVoltage(_adcConverter.Read(_pinNumber)));
+        }
+
+        public int CalculateFsrResistance(int fsrVoltage)
```

Comment:
Same comment than with Voltage, I would probably just remove this one and only expose the ReadResistance method. Also, please remove `Fsr` from both the name and the parameter.